nestkernel/simulation_manager.cpp

```diff
@@ -813,27 +842,6 @@ nest::SimulationManager::update_()
 }
```

Comment:
I don't think so. It is outside of the do-while-loop within `update_()`. Thus it is called only once at the end of `run()` in the changed code. (In addition, the update of synaptic elements also happens within the do-while-loop, but there is this extra call at the end outside of the loop, and this concerns me slightly - at least it is something to consider.)